pages/bookings/index.tsx

```diff
@@ -1,17 +1,20 @@
+import Loader from "@components/Loader";
 import prisma from "@lib/prisma";
-import { useSession } from "next-auth/client";
 import Shell from "@components/Shell";
-import { useRouter } from "next/router";
-import dayjs from "dayjs";
-import { Fragment } from "react";
 import { Menu, Transition } from "@headlessui/react";
+import { ClockIcon, XIcon } from "@heroicons/react/outline";
 import { DotsHorizontalIcon } from "@heroicons/react/solid";
 import classNames from "@lib/classNames";
-import { ClockIcon, XIcon } from "@heroicons/react/outline";
-import Loader from "@components/Loader";
+import dayjs from "dayjs";
+import { GetServerSidePropsContext, InferGetServerSidePropsType } from "next";
+import { useSession } from "next-auth/client";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { Fragment } from "react";
 import { getSession } from "@lib/auth";
 
-export default function Bookings({ bookings }) {
+
+export default function Bookings({ bookings }: InferGetServerSidePropsType<typeof getServerSideProps>) {
```

Comment:
Use `inferSSRProps`, it's there for a reason - https://github.com/vercel/next.js/issues/15913

```suggestion
export default function Bookings({ bookings }: inferSSRProps<typeof getServerSideProps>) {
```